docs/usage.md

```diff
@@ -142,16 +143,17 @@ nf-core/taxprofiler will automatically decompress and extract any compressed arc
 
 The (uncompressed) database paths (`db_path`) for each tool are expected to contain:
 
-- [**Bracken**:](usage/tutorials#bracken-custom-database) output of the combined `kraken2-build` and `bracken-build` process.
-- [**Centrifuge**:](usage/tutorials#centrifuge-custom-database) output of `centrifuge-build`.
-- [**DIAMOND**:](usage/tutorials#diamond-custom-database) output of `diamond makedb`.
-- [**Kaiju**:](usage/tutorials#kaiju-custom-database) output of `kaiju-makedb`.
-- [**Kraken2**:](usage/tutorials#kraken2-custom-database) output of `kraken2-build` command(s).
-- [**KrakenUniq**:](usage/tutorials#krakenuniq-custom-database) output of `krakenuniq-build` command(s).
-- [**MALT**](usage/tutorials#malt-custom-database) output of `malt-build`.
-- [**MetaPhlAn**:](usage/tutorials#metaphlan-custom-database) output of with `metaphlan --install` or downloaded from links on the [MetaPhlAn wiki](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-4#customizing-the-database).
-- [**mOTUs**:](usage/tutorials#motus-custom-database) the directory `db_mOTU/` that is downloaded via `motus downloadDB`.
-- [**ganon**:](usage/tutorials#ganon-custom-database) output of `ganon build` or `ganon build-custom`.
+- [**Bracken**:](usage/tutorials.md#bracken-custom-database) output of the combined `kraken2-build` and `bracken-build` process.
+- [**Centrifuge**:](usage/tutorials.md#centrifuge-custom-database) output of `centrifuge-build`.
+- [**DIAMOND**:](usage/tutorials.md#diamond-custom-database) output of `diamond makedb`.
+- [**Kaiju**:](usage/tutorials.md#kaiju-custom-database) output of `kaiju-makedb`.
+- [**Kraken2**:](usage/tutorials.md#kraken2-custom-database) output of `kraken2-build` command(s).
+- [**KrakenUniq**:](usage/tutorials.md#krakenuniq-custom-database) output of `krakenuniq-build` command(s).
+- [**MALT**](usage/tutorials.md#malt-custom-database) output of `malt-build`.
+- [**MetaPhlAn**:](usage/tutorials.md#metaphlan-custom-database) output of with `metaphlan --install` or downloaded from links on the [MetaPhlAn wiki](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-4#customizing-the-database).
+- [**mOTUs**:](usage/tutorials.md#motus-custom-database) the directory `db_mOTU/` that is downloaded via `motus downloadDB`.
+- [**ganon**:](usage/tutorials.md#ganon-custom-database) output of `ganon build` or `ganon build-custom`.
+- [**kmcp**:](usage/tutorials.md#kmcp-custom-database) output of `kmcp index`. `kmcp index` in each turn is the output of `kmcp compute` step.
```

Comment:
```suggestion
- [**KMCP**:](usage/tutorials.md#kmcp-custom-database) output of `kmcp index`. `kmcp index` in each turn is the output of `kmcp compute` step.
```

Reply:
I don't understand the second sentence here... 'in each turn'?

Reply:
I will remove this, bad translation to english

Reply:
Do you mean index comes in turn after compute?

Reply:
Yes, this is what I meant 😬